install-dev/data/xml/hook.xml

```diff
@@ -11,6 +11,11 @@
       <title>New orders</title>
       <description/>
     </hook>
+    <hook id="actionValidateOrderAfter">
+      <name>actionValidateOrderAfter</name>
+      <title>New orders</title>
```

Comment:
Not sure News orders is the right name as it relies on validate order statement :thinking:

Reply:
validateOrder is only called at order creation time, the are other hooks for order state changes

Reply:
The problem with this sentence is, when you're calling a ` validateOrder` hook, you're not thinking about a hook which have a description telling it happens only when the new order is complete otherwise the action name should be ` actionOnCompleteOrderCreation`  or ` actionOrderComplete` :thinking:
What if this method is called by module outside the order creation?

Reply:
Indeed, but actionValidateOrder hook is already named "New orders"

Reply:
if i change for ?
<hook id="actionOnCompleteOrderCreation">
      <name>actionOnCompleteOrderCreation</name>
      <title>After complete creation of an order</title>
      <description>This hook is called after the complete creation of an order</description>
    </hook>

Reply:
I can change the title "After create complet order" ?

Reply:
`After validating an order` we have to think about method behavior, not the context it is used inside the Core